cmd/mqtt/main.go

```diff
@@ -130,6 +136,22 @@ func main() {
 
 	cc := thingsapi.NewClient(conn, thingsTracer, cfg.thingsAuthTimeout)
 
+	ps, err := nats.NewPubSub(cfg.natsURL, "mqtt", logger)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Failed to connect to NATS: %s", err))
+		os.Exit(1)
+	}
+	defer ps.Close()
+	mqttPub, err := mqttpub.NewPublisher(fmt.Sprintf("%s:%s", cfg.mqttTargetHost, cfg.mqttTargetPort), cfg.mqttForwarderTimeout)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Failed to create MQTT publisher: %s", err))
+		os.Exit(1)
+	}
+	if err := mqtt.Forward(nats.SubjectAllChannels, ps, mqttPub); err != nil {
```

Comment:
`nats.SubjectAllChannels` should be used as arg here `nats.NewPubSub(cfg.natsURL, "mqtt", logger)`